### PR TITLE
Add local solr setup to docs

### PR DESCRIPTION
**Pull request by phjudge**
Improving README.md, including:
- instructions for local solr setup
- instructions for using docker-compose with the app

re: datamade#334

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,10 +55,83 @@ Perform the following steps from your terminal.
...
+    ```
+6. Run the app. In separate terminal windows:
+    ```bash
+    # Launch services listed in docker-compose.yml
+    #
+    # To launch just one:
+    #   docker-compose up <service-name>
+    #
+    # To exclude one from launching:
+    #   docker-compose up --scale <service-name>=0
```
Comment (hancush) [inline README.md]: i prefer the explicit `docker-compose up <service-name> [<service-name> <service-name> ...]` to this override to launch one or more services. what do you think?

File: README.md [hunk 2]
```diff
@@ -48,10 +55,83 @@ Perform the following steps from your terminal.
...
+
+    ```bash
+    mkvirtualenv bga
+    ```
+3. Install the requirements.
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+4. Copy the local settings file. Note that you may need to update `DATABASES`, `REDIS_PORT` and `SOLR_URL` in your own copy of `local_settings.py` to reflect your local setup.
```
Comment (hancush) [inline README.md]:
```suggestion
4. Copy the local settings file. Note that you may need to update `DATABASES`, `REDIS_PORT` and `SOLR_URL` in your own copy of `local_settings.py` to reflect the host port each service is mapped to in `docker-compose.yml`.
```

File: README.md (hunk 1 above)
Comment (phjudge) [reply README.md]: I think that's much better. Resolved in 4ad169f.

File: README.md (hunk 2 above)
Comment (phjudge) [reply README.md]: Applied in 1632a69

**Comment by phjudge** [comment]
@hancush fyi I don't have write access, so you'll need to merge this.

**Pull request merged**